README.rst

```diff
@@ -0,0 +1,7 @@
+About
+-----
+
+This branch of pyWWT hosts data files for sky surveys. At the moment, we only 
```

Comment:
Maybe say 'footprints'? Note that this branch also stored the JS files required by PyWWT, so it might be worth mentioning.

Reply:
"Footprints" in place of "data files"?

Reply:
Maybe it should say 'This branch of the PyWWT repository hosts data files needed by the pywwt package, including instrument footprints as well as javascript files'.